packages/site/.eslintrc.cjs

```diff
@@ -8,7 +8,7 @@ module.exports = {
     "import/no-unresolved": [
       "error",
       {
-        ignore: ["^@atlantis/docs"],
+        ignore: ["^@atlantis/docs", "^@atlantis/packages"],
```

Comment:
The storybook files were erroring because of the way they are importing the readme files. Since thats all they seem to do, I decided to just point the packages pages directly to the readmes